### PR TITLE
[HOPSWORKS-1562] Databricks feature store improvements

### PR DESCRIPTION
**Pull request by jimdowling**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1562

### CODE REVIEW

File: docs/featurestore/featurestore.rst [hunk 1]
```diff
@@ -829,12 +831,13 @@ Connecting to the Feature Store from a Databricks cluster requires a Feature Sto
...
 
-There exist two helper libraries: hopsworks-cloud-sdk and hops-util-py. 
-Hopsworks-cloud-sdk is easier to set up but does not provide some advanced functionalities present in hops-util-py. If you intend to use hops-util-py follow the instructions in "Setting up roles and API keys" then jump directly to "Installing hops-util-py".
+There exist two helper libraries: hopsworks-cloud-sdk and hops. 
+Hopsworks-cloud-sdk does not require a Spark environment (and is easier to set up) but does not provide some advanced spark-based functionality present in hops. If you intend to use the hops library follow the instructions in "Setting up roles and API keys" then jump directly to "Installing hops library".
 
 **Setting up roles and API keys**
 
-Follow the steps described in `Connecting from Amazon SageMaker`_ for setting up Hopsworks API keys and AWS roles and access to secrets. Ensure to use the role that is specified in the *Advanced Options* when creating a Spark cluster in Databricks.
+You also need to ensure to use the role that is specified in the *Advanced Options* when creating a Spark cluster in Databricks.
```
Comment (berthoug) [inline docs/featurestore/featurestore.rst]: Having the first sentence of the first paragraph of the section starting wit "also need" seems strange to me. The all sentence seems out of place to me and I'm not sure the user will easily understand what they need to do.

File: docs/featurestore/featurestore.rst [hunk 2]
```diff
@@ -857,17 +868,17 @@ In the Databricks notebooks connected to the prepared cluster use the following
...
 
-**Installing hops-util-py**
+**Installing hops library**
 
 .. warning:: 
  - This is an advanced setup, if you are not sure that you need it or if you are not sure of what you are doing, please use hopsworks-cloud-sdk. 
  - Hopsworks-cloud-sdk needs to be able to connect directly to the ip on which the hops name node and hive metastore are listening. This means that if you deployed Hopsworks on AWS you will need to set up `VPC Peering <https://docs.databricks.com/administration-guide/cloud-configurations/aws/vpc-peering.html>`_ between your Databricks cluster and the Hopsworks one.
 
-In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries*. Make sure that hopsworks-cloud-sdk is not installed, make sure to uninstall it if that's the case. Then *Install New*. As *Library Source* choose *PyPI* and fill in *hops-util-py* into the *Package* field.
+In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries*. Make sure that hopsworks-cloud-sdk is not installed, make sure to uninstall it if that's the case. Then *Install New*. As *Library Source* choose *PyPI* and fill in *hops* into the *Package* field.
```
Comment (berthoug) [inline docs/featurestore/featurestore.rst]: has the library been renamed? I needed to use hops-util-py when I developed and tested my changes.

Comment (jimdowling) [reply docs/featurestore/featurestore.rst]: it's called 'hops', not hops-util-py. The repo is called hops-util-py for some reason unbeknownst to me.

File: docs/featurestore/featurestore.rst (hunk 1 above)
Comment (jimdowling) [reply docs/featurestore/featurestore.rst]: line 839 is out of place, will fix.

**Comment by jimdowling** [comment]
I think the picture adds value. It is a problem both I and a trial user had - matching up API calls to information in hopsworks that may not be obvious. It's not worse than many of the similar screenshots we have in the docs.

**Comment by jimdowling** [comment]
I fixed the picture. Looks better.

**Pull request merged**